src/routes/Dataset/components/DatasetTechnicalInfo/DatasetTechnicalInfo.js

```diff
@@ -31,8 +29,17 @@ const DatasetTechnicalInfo = ({ dataset, status, t, i18n }) => {
     defaultValue: type
   })
 
-  const createDate = creationDate
-    ? moment(creationDate).format('DD/MM/YYYY')
+  const { creationDate, revisionDate } = dataset.metadata
+
+  const createdAt = creationDate
+  const updatedAt = revisionDate || creationDate
+
+  const createdAtLabel = createdAt
```

Comment:
why not `creationDate ? …` and drop the `createdAt` variable?

Reply:
You're true! It's stupid.